tests/__snapshots__/test_utils/test_bom_generation_altium_with_device_sheets.csv

```diff
@@ -1,39 +1,14 @@
 Comment,Designator,Name,Quantity
-61300211121,P4,Header_Male_1R_02P,1
-61300211121,P3,Header_Male_1R_02P,1
-61300211121,P7,Header_Male_1R_02P,1
-61300211121,P6,Header_Male_1R_02P,1
-61300211121,P1,Header_Male_1R_02P,1
-61300211121,P8,Header_Male_1R_02P,1
-61300211121,P2,Header_Male_1R_02P,1
-61300211121,P5,Header_Male_1R_02P,1
+61300211121,"P4, P3, P7, P6, P1, P8, P2, P5",Header_Male_1R_02P,8
 LM2678SX-3.3,U1,CMP-0069-00188-2,1
-SBR1045D1-13,D1,3c54d775813915f30481cfffa5e9369,1
-UUD1C331MNL1GS,C4,NIC-CAP-P-2,1
-GRM033R61A103KA01J,C1,b4654b650e69147ec39a6b967a45e02,1
-GRM155R61H104KE19D,C3,02cd4be3c70576c341d710602a673aa,1
-IHLP3232DZER100M01,L1,725321dda6339637d4d059423bc1c3c,1
-EEE-1HA470XP,C2,a00bbbee19879c290e62620ae7e47b1,1
-LM2678SX-12,U1,CMP-0069-00189-2,1
-SBR1045D1-13,D1,3c54d775813915f30481cfffa5e9369,1
-UUD1C331MNL1GS,C4,NIC-CAP-P-2,1
-GRM033R61A103KA01J,C1,b4654b650e69147ec39a6b967a45e02,1
-GRM155R61H104KE19D,C3,02cd4be3c70576c341d710602a673aa,1
-IHLP3232DZER100M01,L1,725321dda6339637d4d059423bc1c3c,1
-EEE-1HA470XP,C2,a00bbbee19879c290e62620ae7e47b1,1
+SBR1045D1-13,"D1, D3, D4, D2",3c54d775813915f30481cfffa5e9369,4
+UUD1C331MNL1GS,"C4, C12, C16, C8",NIC-CAP-P-2,4
+GRM033R61A103KA01J,"C1, C9, C13, C5",b4654b650e69147ec39a6b967a45e02,4
+GRM155R61H104KE19D,"C3, C11, C15, C7",02cd4be3c70576c341d710602a673aa,4
+IHLP3232DZER100M01,"L2, L1, L4, L3",725321dda6339637d4d059423bc1c3c,4
+EEE-1HA470XP,"C2, C10, C14, C6",a00bbbee19879c290e62620ae7e47b1,4
+LM2678SX-12,U3,CMP-0069-00189-2,1
 RC0805FR-071KL,R?,RES-2,1
```

Comment:
One thing to note is that we're not substituting these unannotated components, but [Altium seems to](https://hub.allspice.io/AllSpiceTests/Altium-Device-Sheet-Usage-Demo/src/branch/main/DCDC%20Regulators%20Breakout/DCDC%20Regulators%20Breakout%20BOM.csv).

Looking at the annotations file I see `LogicalDesignator21=R2` & `PhysicalDesignator21=R2`, so either Altium is quietly annotating these when it exports the BOM or perhaps @polypour the file you have locally is out of sync with what is pushed to remote. Can you double check? The [sub sheet](https://hub.allspice.io/AllSpiceTests/Altium-Device-Sheets/src/branch/main/PSU-DCDC-ADJ-9V0-TI-LM2678SX.SchDoc) has `R?`, but that doesn't show up in the annotations file.

Reply:
The design review in the AllSpice-Demos parent repo for the mirror in AllSpiceTests was left open and didn't get merged to main, hence the R? refdes. I just merged it. @shrik450, could you please give it a re-run.

Reply:
https://hub.allspice.io/AllSpice-Demos/Altium-Device-Sheets/pulls/3